.github/workflows/test-e2e.yml

```diff
@@ -276,6 +276,8 @@ jobs:
           MCLI_PRIVATE_API_KEY: ${{ secrets.ATLAS_PRIVATE_KEY }}
           MCLI_ORG_ID: ${{ secrets.ATLAS_ORG_ID}}
           MCLI_OPS_MANAGER_URL: "https://cloud-qa.mongodb.com/"
+          ATLAS_LOCAL_PUBLIC_KEY: ${{ secrets.ATLAS_LOCAL_PUBLIC_KEY }}
+          ATLAS_LOCAL_PRIVATE_KEY: ${{ secrets.ATLAS_LOCAL_PRIVATE_KEY }}
```

Comment:
@josvazg  Why do we need to duplicate API credentials?

Reply:
You are right, we do not need this credentials.

I can simplify, but then the gov test issues will get me, so I won't be able to merge today.

Reply:
removed